src/layouts/PlanLayout.astro

```diff
@@ -8,9 +10,12 @@ const {title} = Astro.props;
 
 <html lang="de">
   <head>
+    <meta charset="UTF-8">
     <title>{title}</title>
   </head>
   <body>
-    <slot />
```

Comment:
You should instead of putting the Components into the Layout itself, put the Layout into the Slot while using the Layout in the page itself. If you're going to put all the content in the Layout anyway, there is no need for the Layout to be a layout